nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -2453,16 +2453,25 @@ The H2 Settings section defines the settings for the H2 database, which keeps tr
 |`nifi.h2.url.append`|This property specifies additional arguments to add to the connection string for the H2 database. The default value should be used and should not be changed. It is: `;LOCK_TIMEOUT=25000;WRITE_DELAY=0;AUTO_SERVER=FALSE`.
 |====
 
-
 === FlowFile Repository
 
 The FlowFile repository keeps track of the attributes and current state of each FlowFile in the system. By default,
 this repository is installed in the same root installation directory as all the other repositories; however, it is advisable
 to configure it on a separate drive if available.
 
+There are currently three implementations of the FlowFile Repository, which are detailed below.
+
+|====
+|*Property*|*Description*
+|`nifi.flowfile.repository.implementation`|The FlowFile Repository implementation. The default value is `org.apache.nifi.controller.repository.WriteAheadFlowFileRepository`.  The other current options are `org.apache.nifi.controller.repository.VolatileFlowFileRepository` and `org.apache.nifi.controller.repository.RocksDBFlowFileRepository`.  *NOTE:* Switching repositories should only be done on an instance with zero FlowFiles, and should only be done with caution.
```

Comment:
Suggest wording change in the Note. "Switching _repository implementations_ should only be done on an instance with zero FlowFiles _in queue_, ..."